media-plugins/alsaequal/alsaequal-0.7.1.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit multilib-minimal
+
+DESCRIPTION="A real-time adjustable equalizer plugin for ALSA"
+HOMEPAGE="https://github.com/bassdr/alsaequal"
+SRC_URI="https://github.com/bassdr/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND=">=media-libs/alsa-lib-1.0.27.2[${MULTILIB_USEDEP}]
```

Comment:
Could you update these requirements? The current lowest version of this in the tree is 1.1*, so this is always fulfilled.
Just put media-libs/alsa-lib[blabla] there.